habitat-baselines/habitat_baselines/config/experiments_hab3/bdp_kinematic_oracle.yaml

```diff
@@ -0,0 +1,20 @@
+# @package _global_
```

Comment:
HI Andrew! thank you for working on this. What is "bdp" in the title? maybe we want to add one linear to talk about this in this yaml.